src/models/searchservice/response/DirectAnswerFieldType.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Type for value of DirectAnswer.
+ *
+ * @public
+ */
+export enum DirectAnswerFieldType {
```

Comment:
Could we call this simply `FieldType`? That's how it's referred to on the back-end. This is really a list of profile field types. We may use it for things besides direct answers in the future.